Fix tabview (#119)
bdlukaa/fluent_ui: CHANGELOG.md

## [3.5.3] - [DD/MM/YYYY]

> - Fix scroll buttons when we have too many tabs in `TabView` ([#92](https://github.com/bdlukaa/fluent_ui/issues/92))

Reviewer: ```suggestion
- Fix scroll buttons when there are too many tabs in `TabView` ([#92](https://github.com/bdlukaa/fluent_ui/issues/92))
```
